manim/mobject/coordinate_systems.py

```diff
@@ -665,16 +746,32 @@ def get_lines_parallel_to_axis(
                     lines2.add(new_line)
         return lines1, lines2
 
-    def get_center_point(self):
+    def get_center_point(self) -> np.ndarray:
+        """Gets the center point from coordinates.
```

Comment:
I'm not sure what are the coordinates you mentioned here. It just returns the origin of the coordinate system 
```suggestion
        """Gets the origin of ``NumberPlane``.
```